src/main/java/thePackmaster/cards/monsterhunterpack/HuntingInstincts.java

```diff
@@ -27,8 +27,8 @@ public void triggerOnGlowCheck() {
     }
 
     @Override
-    public void applyPowers() {
-        super.applyPowers();
+    public void applyPowersToBlock() {
```

Comment:
The issue with the previous code was that `applyPowers` was changing the block, but `calculateCardDamage` wasn't. Since `calculateCardDamage` overwrites damage/block values when a card either acquires a target or is played, this meant that the numbers showed up wrong when the card acquired a target (and it's why the use action needed a `hasHuntTarget` check).

Overriding both `applyPowers` and `calculateCardDamage` would also fix this, but the nice thing about `applyPowersToBlock` is that it gets called by both those methods. Convenient if you're doing something that only affects block.